aws/resources/route53_hostedzone.go

```diff
@@ -12,21 +12,33 @@ import (
 )
 
 func (r *Route53HostedZone) getAll(_ context.Context, configObj config.Config) ([]*string, error) {
-	var ids []*string
+	var (
+		ids    []*string
+		marker *string
+	)
+
+	for {
```

Comment:
This one could also use a paginator [NewListHostedZonesPaginator](https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/route53#NewListHostedZonesPaginator)

Reply:
I have already updated the PR.